bigframes/core/indexes/index.py

```diff
@@ -149,6 +151,27 @@ def has_duplicates(self) -> bool:
     def _block(self) -> blocks.Block:
         return self._data._get_block()
 
+    @property
+    def T(self) -> Index:
+        return self.transpose()
+
+    def transpose(self) -> Index:
```

Comment:
Transpose is same as the input?

Reply:
Yeah, Transpose is just a no-op for Series and Index. Added for completness' sake, but not something that is terribly useful.